docs/builds/guides/integration/configuration.md

```diff
@@ -140,13 +140,16 @@ You can use the extended toolbar configuration format to access additional optio
 ```js
     toolbar: {
         items: [ 'bold', 'italic', '|', 'undo', 'redo', '-', 'numberedList', 'bulletedList' ],
+        removeItems: [ 'link' ],
```

Comment:
There should be spaces in the examples? I had to override my editor settings, because, by default, we use tabs.

Reply:
Yup, we use tabs. So perhaps that was misformatted in the past (no eslint for docs unfortunately).